src/datetime/mod.rs

```diff
@@ -200,13 +201,12 @@ impl<Tz: TimeZone> DateTime<Tz> {
     /// # Example
     ///
     /// ```
-    /// use chrono::Utc;
-    /// use chrono::TimeZone;
+    /// use chrono::{Utc, TimeZone, NaiveDate};
     ///
-    /// let dt = Utc.ymd_opt(1970, 1, 1).unwrap().and_hms_milli_opt(0, 0, 1, 444).unwrap();
+    /// let dt = Utc.from_local_datetime(&NaiveDate::from_ymd_opt(1970, 1, 1).unwrap().and_hms_milli_opt(0, 0, 1, 444).unwrap()).unwrap();
```

Comment:
potentially we could consider using `and_timezone(Utc)`, if we prefer to encourage the use of that API?

Reply:
Yes, changed ~all the instances of this pattern.